lib/ui/pages/score/score_page.dart

```diff
@@ -375,39 +370,46 @@ class _ScoreViewerPageState extends State<ScoreViewerPage> with TickerProviderSt
     );
   }
 
-  Widget _buildOneLineCourse(String name, String openClass) {
-    return Container(
-      padding: const EdgeInsets.all(5),
-      child: Row(
-        children: <Widget>[
-          Expanded(
-            child: Text(name),
-          ),
-          Text(openClass)
-        ],
-      ),
-    );
-  }
+  Widget _buildOneLineCourse(String name, String openClass) => Padding(
+        padding: const EdgeInsets.all(5),
+        child: Row(
+          children: [
+            Expanded(
+              child: Text(name),
+            ),
+            Text(openClass)
```

Comment:
Suggest adding a trailing comma better for formatting.

```suggestion
            Text(openClass),
```